Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -0,0 +1,270 @@
+        /// </summary>
+        public MenuItem()
+        {
+            DefaultStyleKey = typeof(MenuItem);
+            IsFocusEngagementEnabled = true;
+        }
+
+        /// <summary>
+        /// This method is used to show the menu for current item
+        /// </summary>
+        public void ShowMenu()
+        {
+            FlyoutButton?.Flyout?.ShowAt(FlyoutButton);
+        }
+
+        /// <summary>
+        /// This method is used to hide the menu for current item
+        /// </summary>
+        public void HideMenu()
+        {
+            FlyoutButton?.Flyout?.Hide();
+        }
+
+        /// <inheritdoc />
+        protected override void OnApplyTemplate()
+        {
+            FlyoutButton = GetTemplateChild(FlyoutButtonName) as Button;
+            _parentMenu = this.FindAscendant<Menu>();
+            IsOpened = false;
+
+            Items.VectorChanged -= Items_VectorChanged;
+
+            if (_menuFlyout == null)
+            {
+                _menuFlyout = new MenuFlyout();
+            }
+            else
+            {
+                _menuFlyout.Opened -= MenuFlyout_Opened;
+                _menuFlyout.Closed -= MenuFlyout_Closed;
+            }
+
+            if (FlyoutButton != null)
+            {
+                FlyoutButton.PointerExited -= FlyoutButton_PointerExited;
+                Items.VectorChanged += Items_VectorChanged;
+
+                _menuFlyout.Placement = _parentMenu.Orientation == Orientation.Horizontal
+                    ? FlyoutPlacementMode.Bottom
+                    : FlyoutPlacementMode.Right;
+
+                _menuFlyout.Opened += MenuFlyout_Opened;
+                _menuFlyout.Closed += MenuFlyout_Closed;
+                FlyoutButton.PointerExited += FlyoutButton_PointerExited;
+
+                _menuFlyout.MenuFlyoutPresenterStyle = _parentMenu.MenuFlyoutStyle;
+                FlyoutButton.Style = _parentMenu.HeaderButtonStyle;
+                ReAddItemsToFlyout();
+
+                FlyoutButton.Flyout = _menuFlyout;
+
+                if (_isAccessKeySupported)
+                {
+                    FlyoutButton.AccessKey = AccessKey;
+                    AccessKey = string.Empty;
+                }
+            }
+
+            base.OnApplyTemplate();
+        }
+
+        internal void ShowTooltip()
+        {
+            var inputGestureText = GetValue(Menu.InputGestureTextProperty) as string;
+            if (string.IsNullOrEmpty(inputGestureText))
+            {
+                return;
+            }
+
+            var tooltip = ToolTipService.GetToolTip(FlyoutButton) as ToolTip;
+            if (tooltip == null)
+            {
+                tooltip = new ToolTip();
+                tooltip.Style = _parentMenu.TooltipStyle;
+                ToolTipService.SetToolTip(FlyoutButton, tooltip);
+            }
+
+            tooltip.Placement = _parentMenu.TooltipPlacement;
+            tooltip.Content = inputGestureText;
```

Comment:
The content should probably have the `Alt+` removed just as the AccessKey tooltip does